scipy/signal/filter_design.py

```diff
@@ -2111,9 +2111,17 @@ def iirdesign(wp, ws, gpass, gstop, analog=False, ftype='ellip', output='ba',
             - Bessel/Thomson: 'bessel'
 
     output : {'ba', 'zpk', 'sos'}, optional
-        Type of output:  numerator/denominator ('ba'), pole-zero ('zpk'), or
-        second-order sections ('sos'). Default is 'ba' for backwards
-        compatibility, but 'sos' should be used for general-purpose filtering.
+        Type of output:  
+            numerator/denominator ('ba'), 
+            pole-zero ('zpk'), 
+            second-order sections ('sos'). 
```

Comment:
When I run the refguide check on this PR, it complains about this section of the docstring.  Sphinx interprets the indentation as a "defintion list", and it expects such a list to be terminated with a blank line.

One way to rework this is really make it a definition list, like this:

```suggestion
        Type of output:

            'ba'
                numerator/denominator transfer function
            'zpk'
                zeros-poles-gain
            'sos'
                second-order sections

```
(The blank line at the end is required--don't leave it out.)

Reply:
@WarrenWeckesser: I hope I followed the spirit of your advice. The file already had several lists, I adapted the style of the list to the form used in the file.

Reply:
How can I run such a refguide check myself?

Reply:
- If you used `runtests.py` to build SciPy locally: `python runtests.py --refguide-check`
- If you used an inplace build: `python tools/refguide_check.py`

Reply:
Thanks! That will be helpful on the next occasion.